src/cargo/ops/cargo_package.rs

```diff
@@ -900,12 +1055,14 @@ fn run_verify(
     ws: &Workspace<'_>,
     pkg: &Package,
     tar: &FileLock,
+    local_reg: Option<&TmpRegistry<'_>>,
     opts: &PackageOpts<'_>,
 ) -> CargoResult<()> {
     let gctx = ws.gctx();
 
     gctx.shell().status("Verifying", pkg)?;
 
+    tar.file().seek(SeekFrom::Start(0))?;
```

Comment:
Why is this needed now when it wasn't before?

Reply:
Previously it was the caller's responsibility. Since we now call `run_verify` twice, I thought it was nicer the move the seek inside.